ui/components/multichain/pages/send/send.js

```diff
@@ -369,6 +369,7 @@ export const SendPage = () => {
       <Header
         textProps={{
           variant: TextVariant.headingSm,
+          as: 'h1',
```

Comment:
Adding the as prop here changes the default `div` to `h1` and improves the semantic html and accessibility of the send page see screenshot in snapshot